common/src/main/scala/explore/components/InputModal.scala

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components
+
+import cats.effect.IO
+import crystal.ViewF
+import crystal.react.implicits._
+import eu.timepit.refined.auto._
+import explore.AppCtx
+import explore.Icons
+import explore.implicits._
+import japgolly.scalajs.react.MonocleReact._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.ui.forms.FormInputEV
+import monocle.macros.Lenses
+import react.common._
+import react.semanticui.elements.button.Button
+import react.semanticui.modules.modal._
+import react.semanticui.sizes.Small
+
+/**
+ * Generic component to accept user input
+ */
+final case class InputModal(
+  title:        String,
+  initialValue: String,
+  label:        String,
+  placeholder:  String,
+  okLabel:      String,
+  trigger:      VdomNode,
+  onComplete:   String => Callback
+) extends ReactProps[InputModal](InputModal.component)
+
+object InputModal {
+  type Props = InputModal
+
+  @Lenses
+  final case class State(inputValue: String)
+
+  object State {
+    val Zero = State("")
```

Comment:
Is this used?

Reply:
Good catch, at some moment state had more fields and unused Zero to initialize but I could remove it now